src/blog/stylo-another-kind-of-rich-text-editor.md

```diff
@@ -0,0 +1,164 @@
+---
+path: "/blog/stylo-another-kind-of-rich-text-editor"
+date: "2022-02-08"
+title: "Stylo - Another Kind Of Rich Text Editor"
+description: "So, I developed a new open source WYSIWYG interactive editor for JavaScript."
+tags: "#javascript #showdev #webdev #webcomponents"
+image: "https://cdn-images-1.medium.com/max/1600/1*kPGH8RB1reGGiq6nU0UaIg.jpeg"
+canonical: "https://zhx6p-eqaaa-aaaai-abbrq-cai.raw.ic0.app/d/converting-svg-to-image-in-javascript"
+---
+
+![](https://cdn-images-1.medium.com/max/1600/1*kPGH8RB1reGGiq6nU0UaIg.jpeg)
+
+*Photo by [Amy Hirschi](https://unsplash.com/@amyhirschi?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Unsplash](https://unsplash.com/s/photos/pen?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)*
+
+*****
+
+So, I developed a new open source WYSIWYG interactive editor for JavaScript. Its goal is to bring great user experience and interactivity to the web, for everyone, with no dependencies. I called it: [Stylo](https://stylojs.com/).
+
+*****
+
+### But Why?
+
+While developing last year the migration of our web editor for slides, [DeckDeckGo](https://deckdeckgo.com), to [DFINITY](https://dfinity.org)’s Internet Computer, we also initialized a new experimental concept to support documents as well.
+
+As a very first test, I wrote and published a blog post on our platform — [Converting SVG To Image (PNG, Webp, etc.) in JavaScript](https://zhx6p-eqaaa-aaaai-abbrq-cai.raw.ic0.app/d/converting-svg-to-image-in-javascript).
+
+Although I was satisfied with the result, I was of course also aware that further enhancements were needed.
+
+That’s why I had the idea to extract the rich text editor capabilities of the core of our application. By limiting the scope of these features, I assumed it would be easier to improve and fix what still needed to be done.
+
+In addition, by publishing the new library as a new open source repo, I was also hoping it would benefit from feedbacks and contributions.
```

Comment:
```suggestion
In addition, by publishing the new library as a new open source repo, I was also hoping it would benefit from community feedback and contributions.
```